internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -762,6 +784,72 @@ resource "azurerm_mssql_server" "test" {
 `, data.RandomInteger, data.Locations.Primary)
 }
 
+func (MsSqlServerResource) password(data acceptance.TestData) string {
```

Comment:
Not knowing much about the code styling of this repositiry, I think this function naming could be improved to something like `mssqlServerResourceWithAadAuthOnly`

Reply:
Thanks, changed to `aadAdminWithAADAuthOnlyWithPassword`.